modules/standard/IO.chpl

```diff
@@ -7722,13 +7797,15 @@ config param ReadBinaryArrayReturnInt = false;
    Note that this routine currently requires a 1D rectangular non-strided array.
 
    :arg data: an array to read into – existing values are overwritten.
-   :arg endian: :type:`ioendian` compile-time argument that specifies the byte order in which
-              to read the numbers. Defaults to ``ioendian.native``.
+   :arg endian: :type:`ioendian` compile-time argument that specifies the byte
+                order to read the numbers in. Defaults to ``ioendian.native``.
```

Comment:
Technically, it's incorrect to end a sentence with a dangling modifier like this.  The original word order avoided the problem.